hoodie-common/src/main/java/com/uber/hoodie/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -97,7 +97,7 @@ protected HoodieActiveTimeline(FileSystem fs, String metaPath, String[] included
     public HoodieActiveTimeline(FileSystem fs, String metaPath) {
         this(fs, metaPath,
             new String[] {COMMIT_EXTENSION, INFLIGHT_COMMIT_EXTENSION, DELTA_COMMIT_EXTENSION,
-                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION,
+                INFLIGHT_DELTA_COMMIT_EXTENSION, COMPACTION_EXTENSION, INFLIGHT_ROLLBACK_EXTENSION, ROLLBACK_EXTENSION,
```

Comment:
Adding instants to the active timeline makes me nervous. We have to be sure this does not impact anything else.